ddtrace/opentelemetry/span.go

```diff
@@ -142,26 +165,35 @@ func (s *span) SetStatus(code otelcodes.Code, description string) {
 // SetAttributes sets the key-value pairs as tags on the span.
 // Every value is propagated as an interface.
```

Comment:
So now the code won't set the attribute as a tag if it is a reserved attribute, correct? That makes sense, but let's update the godoc to clarify this. It would be a good idea to name every single reserved attribute, so users know what to expect